.pre-commit-config.yaml

```diff
@@ -33,7 +33,7 @@ repos:
     hooks:
     - id: flake8
       name: flake8
-      args: ["--max-line-length=150"]
+      args: ["--max-line-length=150", "--per-file-ignores=__init__.py:F401"]
```

Comment:
Love this - we should drop it in `src/server/backend/__init__.py` and do the same for Further Link and the SDK